src/main/java/fi/helsinki/cs/tmc/core/commands/DownloadModelSolution.java

```diff
@@ -0,0 +1,64 @@
+package fi.helsinki.cs.tmc.core.commands;
+
+import com.google.common.base.Optional;
+import com.google.common.base.Strings;
```

Comment:
We sort alphabetically and group by the first few package names.
And our own packages are always first